src/schema/v2/partner.ts

```diff
@@ -194,8 +194,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           },
         },
         resolve: ({ id }, options, { partnerLocationsConnectionLoader }) => {
-          return partnerLocationsConnectionLoader(id, options).then(
-            locations => {
+          return partnerLocationsConnectionLoader(id, {
+            total_count: true,
+            ...options,
+          }).then(locations => {
+            if (locations.body) {
```

Comment:
You don't need this extra defensiveness anymore with the `total_count` param.

Reply:
Fixed this